pkg/log/handler.go

```diff
@@ -307,3 +312,28 @@ func (g *groupOrAttrs) WithAttrs(attrs []slog.Attr) *groupOrAttrs {
 		next:  g,
 	}
 }
+
+// getTraceAttr 组合成单个attr
+func getTraceAttr(ctx context.Context) slog.Attr {
+	spanCtx := trace.SpanContextFromContext(ctx)
+	if !spanCtx.HasTraceID() {
+		return slog.Attr{}
+	}
+
+	traceID := spanCtx.TraceID().String()
+	spans, ok := ctx.Value(spanCtxKey).([]trace.Span)
+	if !ok || len(spans) == 0 {
+		return RidAttr(traceID)
+	}
+
+	// 完整示例较长 rid=660bb87ee3d3883914598c4b95be5910/46e52288b38dbea7/c404c158283e3f99
+	// 使用简短日志 rid=660bb87ee3d3883914598c4b95be5910/46e522/c404c1
+	shortSpans := lo.Map(spans, func(span trace.Span, _ int) string {
+		spanID := span.SpanContext().SpanID().String()
+		// span规范, 16长度, 日志只打印前6位
+		return spanID[:6]
+	})
+	rid := traceID + "/" + strings.Join(shortSpans, "/")
+
+	return RidAttr(rid)
+}
```

Comment:
这个很频繁，每一条日志打印都涉及，建议改用builder
```suggestion
// getTraceAttr 组合成单个attr
func getTraceAttr(ctx context.Context) slog.Attr {
	spanCtx := trace.SpanContextFromContext(ctx)
	if !spanCtx.HasTraceID() {
		return slog.Attr{}
	}

	rid := strings.Builder{}
	rid.Grow(len(traceID) + len(spans)*7)
	rid.WriteString(traceID)
	// 完整示例较长 rid=660bb87ee3d3883914598c4b95be5910/46e52288b38dbea7/c404c158283e3f99
	// 使用简短日志 rid=660bb87ee3d3883914598c4b95be5910/46e522/c404c1
	for i := range spans {
		rid.WriteByte('/')
		spanID := spans[i].SpanContext().SpanID().String()
		rid.WriteString(spanID[:6])
	}

	return RidAttr(rid.String())
}
```

Reply:
那是否要再加个sync.Pool